File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 3/ex7.c
#include <stdio.h>

main()
{
    printf("Numeros entre 1000 e 1999, que, divididos por 9, restam 6\n\n");

    //vars
    int i;

    //Processamento
    for(i=1000; i<=1999; i++)
    {
        if( i%9 == 6)
        {
            printf("%i <- Divisivel por 9 e resta 6\n", i);
        }
        else
        {
            printf("%i\n", i);
        }

    }

}

File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 3/src/Ex7.java
//Importando Scanner
import java.util.Scanner;

public class Ex7 {

	public static void main(String[] args){
		
		//Instanciando Scanner
		Scanner ent = new Scanner(System.in);
		
		System.out.println("Números que divididos por 9 têm resto igual a 6\n\n");
		
		//vars
		int num;
		
		for(num=1000; num<=1999; num++)
		{
			if( num%9 == 6 )
				System.out.println(num + " <- divisível por 9 e resta 6");
			else
				System.out.println(num);
		}
		
	}
}
